.golangci.yml

```diff
@@ -33,8 +33,11 @@ linters:
     - lll
     - mnd
     - nestif
+    - perfsprint
     - prealloc
     - revive
+    - wrapcheck
+    - wsl # deprecated, eventually we'll enable wsl_v5
```

Comment:
maybe a typo?

<img width="1024" height="188" alt="Image" src="https://github.com/user-attachments/assets/f01aaf3c-101f-489f-bdd3-84c35535b00c" />

Reply:
not a typo but probably we can remove. I was just comparing to other config file I have and because I use default: all I had to disable "old" wsl. will address when I go look at the deep source stuff!